bin/scripts/lib/fonts.json

```diff
@@ -70,6 +70,20 @@
       "repoRelease": true,
       "description": "Metrically similar to Arial, pan-European WGL character set, sans serif"
     },
+    {
+      "unpatchedName": "Atkinson Hyperlegible Mono",
+      "licenseId": "OFL-1.1-RFN",
+      "RFN": true,
+      "version": "1.0",
+      "patchedName": "AtkynsonMono",
+      "folderName": "AtkinsonMono",
+      "imagePreviewFont": "AtkynsonMono Nerd Font",
```

Comment:
Is this not `AtkynsonMono NF` :thinking:

Reply:
Again, wasn't sure about where to get these names or information from. Will update :+1: